### PR TITLE
Added spring security, roles, JWTtokens

### PR DESCRIPTION
**Pull request by Djubabagdad**
God save us

### CODE REVIEW

File: src/test/java/test/NewTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.Kh033Java.travelbook.model.User;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.neo4j.ogm.config.Configuration;
+import org.neo4j.ogm.session.SessionFactory;
+
+public class NewTest {
+
```
Comment (vanilmalik) [inline src/test/java/test/NewTest.java]: Give more meaningful name, do refactor

File: src/main/java/com/Kh033Java/travelbook/Application.java [hunk 2]
```diff
@@ -4,10 +4,17 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.Scanner;
+
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/Application.java]: why this is here?

File: src/main/java/com/Kh033Java/travelbook/Application.java [hunk 3]
```diff
@@ -4,10 +4,17 @@
...
 @SpringBootApplication(exclude = {SecurityAutoConfiguration.class})
 public class Application {
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/Application.java]: change name to TravelBookApplication

File: src/main/java/com/Kh033Java/travelbook/validation/ValidationUtil.java [hunk 4]
```diff
@@ -8,9 +8,9 @@
 
     private static String template = "%s with such id [%s] not found";
 
-    public static <T> T checkBeforeGet(final Optional<T> obj, final Long id, final Class clazz) {
-        if (obj.isEmpty()) {
-            throw new NotFoundException(String.format(template, clazz.getSimpleName(), id));
+    public static <T> T checkBeforeGet(final Optional<T> obj, final String login, final Class clazz) {
+        if (obj.isPresent()) {
+            throw new NotFoundException(String.format(template, clazz.getSimpleName(), login));
```
Comment (vanilmalik) [inline src/main/java/com/Kh033Java/travelbook/validation/ValidationUtil.java:13]: this method and class is never used

Comment (Djubabagdad) [reply src/main/java/com/Kh033Java/travelbook/validation/ValidationUtil.java:13]: We need finally clarify about this class, I hope that we will resolve this problem in nearest future.

File: src/main/java/com/Kh033Java/travelbook/Application.java (hunk 3 above)
Comment (Djubabagdad) [reply src/main/java/com/Kh033Java/travelbook/Application.java]: Done

File: src/main/java/com/Kh033Java/travelbook/Application.java (hunk 2 above)
Comment (Djubabagdad) [reply src/main/java/com/Kh033Java/travelbook/Application.java]: Deleted!

File: src/test/java/test/NewTest.java (hunk 1 above)
Comment (Djubabagdad) [reply src/test/java/test/NewTest.java]: Deleted!

**Pull request merged**